### DISCUSSION TITLE
Is my Pinecone set up correctly for Flowise?

### DISCUSSION THREAD

**Discussion by pealingthunder** [FlowiseAI/Flowise#465] [Q&A]
I would like to know if I have set up the PineCone configuration correctly.

As I have been having trouble receiving data from Pinecone.  
I know I used the API Key, Environment, and Index accurately, because I can send data to Pincone.

Can someone confirm if this below is correct?

![image](https://github.com/FlowiseAI/Flowise/assets/138264151/971745f2-874d-4fa6-bf06-a54e7ed3e67a)

Additionally, I have my Flowise deployed on Render.

I appreciate any help to get me up and running.

Thanks, Ray

**Comment by HenryHengZJ** [comment]
This looks right 👍🏻

**Comment by byte-rose** [comment]
@pealingthunder Render free option is not stable, Deploy on AWS
